packages/next-swc/Cargo.lock

```diff
@@ -1843,24 +1830,24 @@ dependencies = [
 
 [[package]]
 name = "swc_ecma_ast"
-version = "0.68.3"
+version = "0.68.4"
```

Comment:
I'm not sure if this is fine, as `0.69.1` has a patch to reduce the binary size

Reply:
I'm all for using the latest versions that result in reduced binary sizes, but used less new versions to get the build to work. See my comment here - https://github.com/vercel/next.js/pull/35127#discussion_r822112081